docs/reference/api.yaml

```diff
@@ -439,6 +439,113 @@ paths:
         - volume
       security:
         - Bearer: []
+  /1.0/pools:
+    get:
+      operationId: PoolList
+      description: List pools.
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: Pools list
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Pool'
+        '204':
+          description: No content
+        '401':
+          description: Unauthorized
+          schema:
+            $ref: '#/definitions/ErrorMessage'
+      tags:
+        - pool
+      security:
+        - Bearer: []
+    post:
+      operationId: PoolCreate
+      description: Creates a pool.
+      parameters:
+        - name: PoolCreateData
+          in: body
+          required: true
+          schema:
+            $ref: '#/definitions/PoolCreateData'
+      consumes:
+        - application/json
+      responses:
+        '201':
+          description: Pool created
+        '400':
+          description: Invalid data
+          schema:
+            $ref: '#/definitions/ErrorMessage'
+        '401':
+          description: Unauthorized
+          schema:
+            $ref: '#/definitions/ErrorMessage'
+        '409':
+          description: Pool already exists
+          schema:
+            $ref: '#/definitions/ErrorMessage'
+      tags:
+        - pool
+      security:
+        - Bearer: []
+  /pools/{pool}:
+    parameters:
+      - name: pool
+        in: path
+        required: true
+        type: string
+    delete:
+      operationId: PoolDelete
+      description: Deletes a pool.
+      responses:
+        '200':
+          description: Pool deleted
+        '401':
+          description: Unauthorized
+          schema:
+            $ref: '#/definitions/ErrorMessage'
```

Comment:
Checking the API code, the `DELETE /1.0/pool{pool}` asserts if there are no more apps in the pool to remove. If it has any, it returns FORBIDDEN (403).

Reply:
done